src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Deprecation.php

```diff
@@ -100,7 +102,7 @@ public function __construct($message, array $trace, $file)
             return;
         }
 
-        $test = isset($line['args'][0]) ? $line['args'][0] : null;
+        $test = $line['args'][0] ?? null;
```

Comment:
Should be reverted.

Reply:
Fabbot threw me an exception.

Reply:
the bridge must remain compatible with PHP 7.1
fabbot doesn't know about that, but we do

Reply:
5.5 actually. 🤓